### PR TITLE
URL(404) fixes for multiple locations 

### PR DESCRIPTION
**Pull request by martinb-bb**
Fixes URLs

### CODE REVIEW

**Review by andrewkenreich — APPROVED** [review]
workinggg

File: website/content/sdk/usage/intros/stocks/fundamental-analysis.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The FA module provides programmatic access to the commands from within the OpenB
 - Financial Modeling Prep
 - Polygon
 
-View [this page](https://docs.openbb.co/terminal/quickstart/api-keys) for a list of all API sources used across the platform.
+View [this page](https://docs.openbb.co/terminal/usage/guides/api-keys) for a list of all API sources used across the platform.
```
Comment (jose-donato) [inline website/content/sdk/usage/intros/stocks/fundamental-analysis.md:14]: this should point to hub

File: website/content/terminal/faqs/general_operation.md [hunk 2]
```diff
@@ -34,15 +34,15 @@ From the Windows Security menu, click on the Firewall & Network Protection tab,
...
+It could be that you are running an outdated version in which the menu or command is not yet available. Please check the [installation guide](https://my.openbb.dev/app/sdk/installation) to download the most recent release.
 
 Do note that it is also possible that the menu or command has been deprecated. If this is oversight, please reach out to us [here](https://openbb.co/support).
 
 </details>
 
 <details><summary>Charts do not display on Linux/WSL or Docker installation.</summary>
 
-Check that X-11, or similar, is installed, open, and configured. Follow the instructions pertaining to the system here: [https://docs.openbb.co/terminal/quickstart/installation#3-configuring-your-x-server-to-show-plots](https://docs.openbb.co/terminal/quickstart/installation#3-configuring-your-x-server-to-show-plots)
+Check that X-11, or similar, is installed, open, and configured. Follow the instructions pertaining to the system here: [https://docs.openbb.co/terminal/installation/docker](https://docs.openbb.co/terminal/installation/docker)
```
Comment (jose-donato) [inline website/content/terminal/faqs/general_operation.md:45]: should also point to hub

File: website/content/sdk/usage/intros/stocks/fundamental-analysis.md (hunk 1 above)
Comment (martinb-bb) [reply website/content/sdk/usage/intros/stocks/fundamental-analysis.md:14]: what are the URLS?

File: website/content/terminal/faqs/general_operation.md (hunk 2 above)
Comment (martinb-bb) [reply website/content/terminal/faqs/general_operation.md:45]: What are urls?

**Comment by martinb-bb** [comment]
@jose-donato  this was auto merged. I will make a quick PR for the new URLS when you can provide me with them.

**Pull request merged**